File: Assignment02/Assignment2_5.c
#include<stdio.h>
#include<stdbool.h>

bool Check(int iNo)
{
    bool bFlag=false;

    if(iNo%2==0)
    {
        bFlag=true;
    }
    else
    {
       bFlag=false;
    }

    return bFlag;
}

int main()
{
    int iValue=0;
    bool bRet=false;

    printf("Enter The Number\n");
    scanf("%d",&iValue);

    bRet=Check(iValue);

    if(bRet==true)
    {
        printf("%d is EVEN",iValue);
    }
    else
    {
        printf("%d is ODD",iValue);
    }

    return 0;
}

File: Python/Assignment2/Assignment2_5.py


def CheckEven(iNum):
    

    if(iNum%2==0):
        return True
    else:
        return False   

def main():
    bRet=False
    iNo=int(input("Enter Number"))
    bRet=CheckEven(iNo)

    if(bRet==True):
        print("It is Prime Number")
    else:
        print("It is Not Prime NUmber")
   

if __name__=="__main__":
    main()